tensilelite/rocisa/rocisa/include/hardware_caps.hpp

```diff
@@ -301,7 +301,7 @@ inline std::map<std::string, bool> initArchCaps(const IsaVersion& isaVersion)
     rv["HasEccHalf"]
         = checkInList(isaVersion, {{9, 0, 6}, {9, 0, 8}, {9, 0, 10}, {9, 4, 2}, {9, 5, 0}});
     rv["Waitcnt0Disabled"] = checkInList(isaVersion, {{9, 0, 8}, {9, 0, 10}, {9, 4, 2}, {9, 5, 0}});
-    rv["HasLDSGT64K"]      = checkInList(isaVersion, {{9, 5, 0}});
+    rv["Has160KLDS"]      = checkInList(isaVersion, {{9, 5, 0}});
```

Comment:
Would you revert this change?
HasLDSGT64K will be removed in #2035.
No need to make change (to avoid conflict).